<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-21T23:24:03 PDT" name="Default suite" finished-at="2024-04-21T23:24:24 PDT" duration-ms="20905">
    <groups>
    </groups>
    <test started-at="2024-04-21T23:24:03 PDT" name="Default test" finished-at="2024-04-21T23:24:24 PDT" duration-ms="20905">
      <class name="com.allianz.test.LoginUITest">
        <test-method signature="applicationDescriptionTest()[pri:1, instance:com.allianz.test.LoginUITest@3cebbb30]" started-at="2024-04-21T23:24:03 PDT" name="applicationDescriptionTest" finished-at="2024-04-21T23:24:15 PDT" duration-ms="11921" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- applicationDescriptionTest -->
        <test-method signature="titleTest()[pri:2, instance:com.allianz.test.LoginUITest@3cebbb30]" started-at="2024-04-21T23:24:15 PDT" name="titleTest" finished-at="2024-04-21T23:24:24 PDT" duration-ms="8839" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [OrangeHRM] but found [OpenEMR]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [OrangeHRM] but found [OpenEMR]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.allianz.test.LoginUITest.titleTest(LoginUITest.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleTest -->
      </class> <!-- com.allianz.test.LoginUITest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
